doc/INSTALL.md

```diff
@@ -276,6 +277,37 @@ need to include `testnet=1`
     ./lightningd/lightningd &
     ./cli/lightning-cli help
 
+To Build on Arch Linux
+---------------------
+
+Install dependencies:
+
+```
+pacman --sync autoconf automake gcc git make python-pip
+pip install --user poetry
+```
+
+Clone Core Lightning:
+
+```
+$ git clone https://github.com/ElementsProject/lightning.git
+$ cd lightning
+```
+
+Build Core Lightning:
+
+```
+~/.local/bin/poetry install
```

Comment:
```suggestion
python -m poetry install
```

This make you sure to run the same instance of python, so you can have no problem different path